### PR TITLE
chore: re-enable thorchain features post hardfork

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

Effectively reverts https://github.com/shapeshift/web/pull/7690/files, to be merged once we're ready to re-enable THORChain after the hardfork.

## Issue (if applicable)

N/A

## Risk

> High Risk PRs Require 2 approvals

Medium - affects all THORChain functionality, so we'll want to check that everything works again.

> What protocols, transaction types, wallets or contract interactions might be affected by this PR?

All THORChain functionality.

## Testing

Check that all THORChain functionality works as expected, including:

- Swapping
- Assets, balances, TX history
- Lending
- LPs
- rFOX staking/unstaking and all related things

### Engineering

☝️

### Operations

- [ ] :checkered_flag: My feature is behind a flag and doesn't require operations testing (yet)

☝️, ➕

The following PRs were not tested by ops in the last release as THORChain was disabled:

```
feat: thorchain LP USDT allowance reset (#7674)
feat: thorchain savers deposit usdt allowance reset step (#7670)
chore: migrate contracts helpers to package (#7612)
```

## Screenshots (if applicable)

N/A

### CODE REVIEW

**Review by 0xApotheosis — COMMENTED** [review]
THORChain functionality after applying the environment config in this diff:

✅ Trades

Screenshot 2024-09-09 at 9 27 01 AM

❓ Getting a curious response attempting to repay a loan

Screenshot 2024-09-09 at 9 29 20 AM

❌  LP remove liquidity

`send` failing when attempting to withdraw

Screenshot 2024-09-09 at 9 52 12 AM

❌ LP add liquidity

'send' failing when attempting to add liquidity

Screenshot 2024-09-09 at 9 53 19 AM

✅ Savers vaults

Deposit is happy

Screenshot 2024-09-09 at 10 09 12 AM

Withdraw it happy

Screenshot 2024-09-09 at 10 09 28 AM

File: .env.base [hunk 1]
```diff
@@ -41,7 +41,7 @@ REACT_APP_FEATURE_CUSTOM_TOKEN_IMPORT=true
 REACT_APP_FEATURE_ARBITRUM_BRIDGE_CLAIMS=true
 REACT_APP_FEATURE_USDT_APPROVAL_RESET=true
 REACT_APP_FEATURE_PORTALS_SWAPPER=true
-REACT_APP_FEATURE_RUNEPOOL=false
+REACT_APP_FEATURE_RUNEPOOL=true
```
Comment (gomesalexandre) [inline .env.base:44]: @0xApotheosis since LP is happy but lending and RUNE things aren't, can we revert this guy and keep for a follow-up?

**Review by gomesalexandre — COMMENTED** [review]
Tested LP locally with https://github.com/shapeshift/web/pull/7510 merged in (though pretty sure it would work just the same without, since the happy paths do not require RUNE Txs). Confirmed all LP things are happy for *non-RUNE Txs*, and borked for Txs involving RUNE (tested with asym RUNE deposits/withdraws, but assuming those will fail just the same for sym, as RUNE Txs do across the board atm).

Not sure how we feel about this one, but I'd err on the side of progression and getting this one in (without the RUNEPool flag) to unbork at least *some* of the functionality of LP.

### THORChain LP

#### Native Asset asym asset Deposits ✅

- Confirmed Add Liquidity Txs are happy for non-RUNE Txs (though requires a fix for native assets following a regression introduced in https://github.com/shapeshift/web/pull/7674/files#diff-6aa5ead536ac54a51d6914e937c1e8c0b3a8bd355e733ed3ff4f36ea04151b0eR560 - authored in https://github.com/shapeshift/web/pull/7705 for viz and cherry-picked to this commit for the sake of getting this one in ASAP). See before the fix. vs. after.
 
  - Before
    Screenshot 2024-09-09 at 12 28 02
  - After

    Screenshot 2024-09-09 at 12 34 55
    Screenshot 2024-09-09 at 12 34 55
    image
    image

#### Token asym asset Deposits ✅

- Confirmed happy: 
    https://jam.dev/c/3f9ed5c9-5d2e-4657-ba2e-38fb3e4dad37

### RUNE asym Deposits 🚫

- Confirmed sad as noted in https://github.com/shapeshift/web/pull/7692#pullrequestreview-2288636094

### Token asym asset withdraws ✅

https://jam.dev/c/7c9a19cc-a291-457e-80f1-94bc8018fa8f

### Native asset asym asset withdraws ✅

https://jam.dev/c/91f8316f-aed4-4edc-83ab-768369859922

### Asym RUNE withdraws 🚫

- Confirmed sad as noted in https://github.com/shapeshift/web/pull/7692#pullrequestreview-2288636094

### Lending

- Unable to test as I don't have any active position, but borked upstream as per last THOR annoucements.

File: .env.base (hunk 1 above)
Comment (gomesalexandre) [reply .env.base:44]: Discussed this one out with @kaladinlight, going to revert RUNEPool from this diff and land this guy 🙏🏽

Comment (gomesalexandre) [reply .env.base]: 91276a3ce504e48de83791a0ae3f09ad0c1e531f

**Review by kaladinlight — APPROVED** [review]
Approving knowing there are to be follow ups for rune pool and mm snaps.

**Pull request merged**